micro_sam/training/semantic_sam_trainer.py

```diff
@@ -107,3 +140,39 @@ def _get_model_outputs(self, batched_inputs):
         # masks = torch.stack([output["masks"].squeeze(0) for output in batched_outputs])
         masks = batched_outputs["masks"]
         return masks
+
+
+class SemanticSamLogger3D(TensorboardLogger):
```

Comment:
This logger should now work for 2d and 3d. Should we rename to `SemanticSamLogger` and then use it for the `SemanticSamTrainer` by default?

Reply:
Yep, makes sense. Updated it in a latest commit.